src/ai-bundle/src/AiBundle.php

```diff
@@ -291,6 +292,26 @@ private function processPlatformConfig(string $type, array $platform, ContainerB
             return;
         }
 
+        if ('cartesia' === $type) {
+            $definition = (new Definition(Platform::class))
+                ->setFactory(CartesiaPlatformFactory::class.'::create')
+                ->setLazy(true)
+                ->addTag('proxy', ['interface' => PlatformInterface::class])
+                ->setArguments([
+                    $platform['api_key'],
+                    $platform['version'],
+                    $platform['host'],
+                    new Reference($platform['http_client'], ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                    new Reference('ai.platform.model_catalog.cartesia'),
+                    new Reference('event_dispatcher'),
```

Comment:
you copied my mistake from #802 which was patched with #831:
```suggestion
                    new Reference('ai.platform.model_catalog.cartesia'),
                    null,
                    new Reference('event_dispatcher'),
```